spec/features/expenses/edits_expense.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+feature "edits expense" do
+  scenario "successfully" do
+    user = create(:user)
+    create(:budget, user: user)
+    log_in_as user
+    expense = create(:expense)
```

Comment:
Can we set the frequency here explicitly? So that we know it definitely changed

Reply:
I would also switch lines 7 and 8, as creating all the objects and then logging in

Reply:
Done in sha: 0648c4a